docs/readthedocs/source/doc/LLM/Quickstart/benchmark_quickstart.md

```diff
@@ -0,0 +1,66 @@
+# Benchmarking
+
+We can do benchmarking on Intel GPUs and CPUs. You can refer to [this](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install.html) to prepare the environment.
+
+## Clone Repo
+
+Navigate to your local workspace and then download BigDL from GitHub.
+
+```
+cd your/local/workspace/
+git clone https://github.com/intel-analytics/BigDL.git
+```
+
+## Configure YAML File
+
+Replace the config.yaml under all-in-one folder with your own YAML file. The format of the content is as follows.
```

Comment:
which "all-in-one folder"?

Reply:
After putting this cd in the clone repo section, it should be clear?
![image](https://github.com/intel-analytics/BigDL/assets/21994046/6af86905-b861-4de8-a8f1-856e9d03970d)

Reply:
`BigDL/python/llm/dev/benchmark/all-in-one`. Will describe it more clearly in next commit.